Assets/Scripts/Game/Entities/PlayerEntity.cs

```diff
@@ -364,12 +368,11 @@ public override void Update(DaggerfallEntityBehaviour sender)
                 if (((i + lastGameMinutes) % 54720) == 0) // 38 days
                 {
                     RegionPowerAndConditionsUpdate(true);
-                    // GetVampireOrWerecreatureQuest(); Non-cure quest
+                    GetVampireOrWereCreatureQuest(false);
                 }
 
-                // TODO: Get vampire/werecreature quest
-                //if (((i + lastGameMinutes) % 120960) == 0) // 84 days
-                //    GetVampireOrWerecreatureQuest(); cure quest
+                if (((i + lastGameMinutes) % 120960) == 0) // 84 days
```

Comment:
I'll just comment this out again temporarily while working on the rest of vampirism. @ajrb might also want to deploy elsewhere via his quest delivery setup. This is an important bit of reference code though, thank you!